### DISCUSSION TITLE
The Beginner WordPress Developer Learning Pathway

### DISCUSSION THREAD

**Discussion by jonathanbossenger** [WordPress/developer-blog-content#287] [Topic Ideas]
In June, the Beginner WordPress Developer learning pathway was published on Learn.WordPress.org.

This article will examine this course in-depth, covering its conception, development process, an overview of each module, future learning pathways, and any other planned improvements.

## Overview:

- Introducing the learning pathway
- A little bit of history behind it
- The process of creating the content
- Overview of the 12 modules
- Future improvements planned
- The next developer focused learning pathways

**Comment by msalihllc** [comment]
Very good
